src/hooks/use-tab-title.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from 'react'
+
+export const useTabTitle = (title: string = 'Deltakeroversikt') => {
+
+	useEffect(() => {
+		document.title = title
+	}, [])
```

Comment:
Ikke så relevant for koden slik det er nå, men det hadde vært greit å sette `title` som en dependency slik at tab tittlen oppdateres når vi endrer `title` dynamisk

Reply:
Jepp, hadde funnet den om jeg hadde kjørt lint:fix... Fikset nå